packages/dd-trace/src/appsec/iast/taint-tracking/rewriter.js

```diff
@@ -50,7 +52,11 @@ function getRewriter (telemetryVerbosity) {
           getGetOriginalPathAndLineFromSourceMapFunction(chainSourceMap, getOriginalPathAndLineFromSourceMap)
       }
 
-      rewriter = new Rewriter({ csiMethods, telemetryVerbosity: getName(telemetryVerbosity), chainSourceMap })
+      rewriter = new Rewriter({
+        csiMethods,
+        telemetryVerbosity: getName(telemetryVerbosity),
+        chainSourceMap
+      })
```

Comment:
Q: Is there any configuration to enable/disable hardcoded secrets detection?

Reply:
@uurien It's worth allowing the detection to be enable/disable?